### ISSUE TITLE
SC1088 Parsing stopped here. Invalid use of parentheses? on a valid Windows Dockerfile

### ISSUE THREAD

**Issue by okainov** [hadolint/hadolint#645]
- [x] This is a bug report
- [ ] This is a feature request
- [X] I searched existing issues before opening this one

Dockerfile:
```
FROM mcr.microsoft.com/dotnet/framework/sdk:4.8

RUN powershell -Command foreach ($file in Get-ChildItem C:\TEMP\certs\*.crt) {Import-Certificate -FilePath $file -CertStoreLocation Cert:\LocalMachine\Root}
```

### Expected behavior

No issues should be found

### Actual behavior
```
-:3 SC1088 error: Parsing stopped here. Invalid use of parentheses?
```
